src/System.Security.Cryptography.Xml/src/System.Security.Cryptography.Xml.csproj

```diff
@@ -90,5 +90,20 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
The auto-declared item that I originally found in the project didn't seem to specify a generator (which is required for strongly-typed resources). Is there a trick to configuring the generator in this scenario?